README.md

```diff
@@ -93,57 +93,116 @@ dicom-anonymizer InputFilePath OutputFilePath --dictionary dictionary.json
+from dicomanonymizer import *
+
+def main():
+    parser = argparse.ArgumentParser(add_help=True)
+    parser.add_argument('input', help='Path to the input dicom file or input directory which contains dicom files')
+    parser.add_argument('output', help='Path to the output dicom file or output directory which will contains dicom files')
+    parser.add_argument('--suffix', action='store', help='Suffix that will be added at the end of series description')
+    args = parser.parse_args()
+
+    input_dicom_path = args.input
+    output_dicom_path = args.output
+
+    extraAnonymizationRules = {}
+
+    def setupSeriesDescription(dataset, tag):
+        element = dataset.get(tag)
+        if element is not None:
+            element.value = element.value + '-' + args.suffix
+
+    # Store all tags into one array and setup the action in
+    # order to keep these value as it is
+    # All tags are defined on file dicomfields.py
+    allTags = []
+    allTags.extend(D_TAGS)
+    allTags.extend(Z_TAGS)
+    allTags.extend(X_TAGS)
+    allTags.extend(U_TAGS)
+    allTags.extend(Z_D_TAGS)
+    allTags.extend(X_Z_TAGS)
+    allTags.extend(X_D_TAGS)
+    allTags.extend(X_Z_D_TAGS)
+    allTags.extend(X_Z_U_STAR_TAGS)
+
+    # the 'keep' method is already defined into the dicom-anonymizer
+    # It will overrides the default behaviour
+    for i in allTags:
+        extraAnonymizationRules[i] = keep
+
+    if args.suffix:
+        extraAnonymizationRules[(0x0008, 0x103E)] = setupSeriesDescription
+
+    # Launch the anonymization
+    anonymize(input_dicom_path, output_dicom_path, extraAnonymizationRules)
+
+if __name__ == "__main__":
+    main()
 ```
 
 In your own file, you'll have to define:
 - Your custom functions. Be careful, your functions always have in inputs a dataset and a tag
-- A dictionary which map your functions to a string
+- A dictionary which map your functions to a tag
+
+## Anonymize dicom tags without dicom file
 
-# Overrides actions for default tags
-You can have the need to keep some tags, for example UID. Then, instead of replace it (`replaceUID` action) then you can override it :
+If for some reason, you need to anonymize dicom fields without initial dicom file (extracted from a database for example). Here is how you can do it:
+```python
+from dicomanonymizer import *
+
+def main():
+
+  # Create a list of tags object that should contains id, type and value
+  fields = [
+    { # Replaced by Anonymized
+      'id': (0x0040, 0xA123),
+      'type': 'LO',
+      'value': 'Annie de la Fontaine',
+    },
+    { # Replaced with empty value
+      'id': (0x0008, 0x0050),
+      'type': 'TM',
+      'value': 'bar',
+    },
+    { # Deleted
+      'id': (0x0018, 0x4000),
+      'type': 'VR',
+      'value': 'foo',
+    }
+  ]
+
+  # Create a readable dataset for pydicom
+  data = pydicom.Dataset()
+
+  # Add each field into the dataset
+  for field in fields:
+    data.add_new(field['id'], field['type'], field['value'])
```

Comment:
alternatively, you could have done `data.from_json(fields)` (nothing to change here, just wanted to let you know :-) )